lib/main.dart

```diff
@@ -19,9 +19,9 @@ class MyApp extends StatelessWidget {
       value: AuthService().user,
       child: MaterialApp(initialRoute: '/screens/wrapper', routes: {
         '/screens/wrapper': (context) => Wrapper(),
-        '/shifts/add-availability': (context) => AddAvailability(),
-        '/shifts/request-shifts': (context) => RequestShift(),
-        '/shifts/view-shifts': (context) => ViewShift(),
+        '/shifts/availability': (context) => Availability(),
+        '/shifts/announcement': (context) => Announcement(),
+        '/shifts/schedule': (context) => Schedule(),
```

Comment:
This might be a stupid question but do we need these? Would it work to just render them like normal widgets within the home screen?